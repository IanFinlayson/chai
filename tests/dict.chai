def main():
    var ages = {"Joe" : 31, "Alice":28, "Olive": 56}
    print(ages["Joe"])
    ages["Joe"] = ages["Joe"] + 1
    print(ages["Joe"])
    ages["Joe"] += 1
    print(ages["Joe"])
    print(ages)
    
