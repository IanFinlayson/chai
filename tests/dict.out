31
32
33
{"Olive": 56, "Joe": 33, "Alice": 28}
